PUBSTWO-1700 (put the report number in the <publisher_item> tag)

orosz-usgs opened the pull request:
In body.ftlx  template, added <publisher_item> tag and removed the
<component> tag as described in the issue.
  Deleted unused Bean numberedSeriesXml() and corresponding xml resource.
  Deleted unused Bean ofr20131259Xml() and corresponding xml resource.
  Updated test result testOneNumberedSeriesPub.xml with the tags changes.

Commit:
PUBSTWO-1700 (put the report number in the <publisher_item> tag)
  In body.ftlx  template, added <publisher_item> tag and removed the
<component> tag as described in the issue.
  Deleted unused Bean numberedSeriesXml() and corresponding xml resource.
  Deleted unused Bean ofr20131259Xml() and corresponding xml resource.
  Updated test result testOneNumberedSeriesPub.xml with the tags changes.

dsteinich reviewed (commented):
Should the header.ftlx be updated to reflect version 4.4.2 also?

orosz-usgs commented:
Good question, I'll confirm with Jim if we are good to update.

orosz-usgs commented:
Jim made PUBSTWO-1717 ticket for the crossref schema version update.

Commit:
PUBSTWO-1700 (put the report number in the <publisher_item> tag)
   Deleted unused Configuration BusServiceConfig and aossciated xml resources
   Deleted unused bean definitions in TestSpringConfig and asscoaited resource.
   Removed TestSpringConfig.class from IT tests

jkreft-usgs commented on src/test/resources/testResult/testOneNumberedSeriesPub.xml:
```diff
@@ -24,6 +24,9 @@
                 <publication_date media_type="online">
                     <year>2013</year>
                 </publication_date>
+                <publisher_item xmlns="http://www.crossref.org/schema/4.4.2">
```
Why is there an extra namespace declaration here?  It should be at the top of the document

orosz-usgs replied:
good catch, taken out.

jkreft-usgs commented on src/main/resources/templates/crossref/body.ftlx:
```diff
@@ -13,6 +13,9 @@
 				<publication_date media_type="online">
 					<year>${pub.publicationYear}</year>
 				</publication_date>
+				<publisher_item xmlns="http://www.crossref.org/schema/4.4.2">
```
Why is there an extra namespace declaration here? It should be at the top of the document

orosz-usgs replied:
good catch, taken out.

Commit:
PUBSTWO-1700 (put the report number in the <publisher_item> tag)
  Removed the namespace declaration from the publisher_item tag.
